code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -82,3 +82,60 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
 			. += span_info("Good work!")
 		if(11 to INFINITY)
 			. += span_info("Incredible!")
+
+/obj/structure/sign/collision_counter
+	name = "incident counter"
+	sign_change_name = "Indicator board- Tram incidents"
+	desc = "A display that indicates how many tram related incidents have occured today."
+	icon_state = "tram_hits"
+	is_editable = TRUE
+	var/hit_count = 0
+	var/tram_id = TRAM_LIFT_ID
+
+/obj/structure/sign/collision_counter/Initialize(mapload)
+	. = ..()
+
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/sign/collision_counter/LateInitialize()
+	. = ..()
+	for(var/obj/structure/industrial_lift/tram/tram as anything in GLOB.lifts)
+		RegisterSignal(tram, COMSIG_TRAM_COLLISION, .proc/new_hit)
```

Comment:
I mean why have the tram_id if it isn't used for anything?
shouldn't you do this
```suggestion
	for(var/obj/structure/industrial_lift/tram/tram as anything in GLOB.lifts)
		if(tram.id != tram_id) // Likely the wrong var, an example
		 	continue
		RegisterSignal(tram, COMSIG_TRAM_COLLISION, .proc/new_hit)
```

Reply:
You're reading the code, what var you want to look at instead of how I had it implemented? Just commit the change.

Reply:
you were doing filtering before no?